src/mmw/apps/modeling/calcs.py

```diff
@@ -110,7 +110,8 @@ def catchment_water_quality(geojson):
           tn_urban_k, tn_riparia, tn_ag_kgyr, tn_natural, tn_pt_kgyr,
           tp_urban_k, tp_riparia, tp_ag_kgyr, tp_natural, tp_pt_kgyr,
           tss_urban_, tss_rip_kg, tss_ag_kgy, tss_natura,
-          tn_yr_avg_, tp_yr_avg_, tss_concmg, ST_AsGeoJSON(geom) as geom
+          tn_yr_avg_, tp_yr_avg_, tss_concmg,
+          ST_AsGeoJSON(ST_Simplify(geom, 0.001)) as geom
```

Comment:
The sample polygon in the image does seem very simplified, is there a balance in this threshold that could get us a little more detail without getting a lot more nodes?